python/tvm/relay/build_module.py

```diff
@@ -193,6 +193,18 @@ def get_params(self):
             ret[key] = value.data
         return ret
 
+@register_func("tvm.relay.build")
+def _rust_build_module(mod, target=None, target_host=None, params=None, mod_name="default"):
+    print(mod)
+    print("\n")
+    rt_mod = build(mod, target, target_host, params, mod_name).module
+    print(rt_mod)
+    print(rt_mod["default"])
```

Comment:
leftover prints, or something we want to keep?

Reply:
leftover, going to try and land this now finally